json-lint/test/resources/schema/inventory/valid/taboola.test.json

```diff
@@ -5,24 +5,40 @@
     },
     "bottom_details_1": {
       "adServer": "taboolaNoStyles"
+    },
+    "bottom_details_3": {
+      "adServer": "taboolaProperties"
     }
   },
   "adServers": {
     "taboola": {
       "type": "taboola",
-      "publisher": "brasil247",
+      "publisher": "marfeel",
       "heights": "(min-width:1565px) 73%, (min-width:1106px) 79%, (min-width:788px) 88%, (min-width:769px) 117%, (min-width:726px) 90%, (min-width:521px) 103%, (min-width:437px) 140%, (min-width:391px) 565%, (min-width:365px) 596%, 624%",
       "article": "auto",
       "mode": "thumbnails-a",
       "placement": "Below Article Thumbnails"
     },
     "taboolaNoStyles": {
       "type": "taboola",
-      "publisher": "brasil247",
+      "publisher": "marfeel",
```

Comment:
Do not **change** tests, **add** tests. 

Changing tests is a way to potentially make invalid something that before was valid, in other words, make invalid that what now we have in production.